### DISCUSSION TITLE
Unable to get the OpenCV plugin to load.

### DISCUSSION THREAD

**Discussion by fbadenhorst5** [koush/scrypted#369] [General]
Extract of what I see in the console (Warning, I'm a noob):

 copying gi\overrides\__init__.py -> build\lib.win-amd64-cpython-310\gi\overrides
  running build_ext
  pycairo: trying include directory: 'C:\\Windows\\Temp\\pip-build-env-b_izi02m\\overlay\\Lib\\site-packages\\cairo\\include'
  pycairo: found 'C:\\Windows\\Temp\\pip-build-env-b_izi02m\\overlay\\Lib\\site-packages\\cairo\\include\\py3cairo.h'
  building 'gi._gi' extension
  error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
  [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for PyGObject
Failed to build PyGObject
ERROR: Could not build wheels for PyGObject, which is required to install pyproject.toml-based projects
WARNING: Ignoring invalid distribution -p (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -p (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -ip (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution - (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -p (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -p (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -ip (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution - (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -p (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -p (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution -ip (c:\python310\lib\site-packages)
WARNING: Ignoring invalid distribution - (c:\python310\lib\site-packages)
pip install result 1

**Comment by koush** [comment]
opencv is not supported on windows. windows is still experimental.
